src/sentry/incidents/logic.py

```diff
@@ -1138,6 +1140,22 @@ def get_alert_rule_trigger_action_msteams_channel_id(organization, integration_i
     return channel_id
 
 
+def get_alert_rule_trigger_action_pagerduty_service_id(organization, integration_id, name):
```

Comment:
@ceorourke I am not sure why we would want this function at all when we are using a dropdown to select the service ID.